site/src/templates/componentPage.tsx

```diff
@@ -13,12 +13,35 @@ import {
   SidebarSection,
   SidebarTab,
 } from "../components/SidebarAndContent"
+import { sortSidebarTabs, stripTrailingSlash } from "./util"
+
+const renderSidebarTabs = (pages, currentPath) => {
+  return pages.map((node, i) => (
+    <SidebarTab
+      href={node!.node!.fields!.slug}
```

Comment:
I realise this wasn't introduce in this PR, but could this `node!.node!.fields!.slug` shenaningens be moved to a variable, so it at least gets a name explain what it is?

Reply:
Actually, it's the `node!.node!` bit that's the hardest to understand, and it seems really susceptible to changes of structure? Perhaps a function like like `getFooNode` (I don't actually know what Foo is to give it a name)

Reply:
> it seems really susceptible to changes of structure?

Yeah 👎 Not sure what you mean by using a function as a way to remedy that though. Wouldn't it be just as susceptible?

Reply:
The `node.node` bit is part of the structure of Gatsby GraphQL queries, not very intuitive.

Reply:
Actually I'll just merge this and raise an issue to clean this up as tech debt. So far we've been pretty good on striking off issues so there's a good chance I'll get to it.

Reply:
Done. See https://github.com/cultureamp/kaizen-design-system/issues/60